tests/framework/lookup/test_lookup.py

```diff
@@ -204,24 +217,87 @@ def test_lookup_table_interpolated_return_types(base_config: LayeredConfigTree)
             "age": (0, 125),
         },
     )
-
-    simulation = InteractiveContext(components=[TestPopulation()], configuration=base_config)
+    component = TestPopulation()
+    simulation = InteractiveContext(components=[component], configuration=base_config)
     manager = simulation._tables
-    table = manager._build_table(data, value_columns="value")(
+    table = manager._build_table(component, data, "", value_columns="value")(
         simulation.get_population_index()
     )
     # make sure a single value column is returned as a series
     assert isinstance(table, pd.Series)
 
     # now add a second value column to make sure the result is a df
     data["value2"] = data.value
-    table = manager._build_table(data, value_columns=["value", "value2"])(
+    table = manager._build_table(component, data, "", value_columns=["value", "value2"])(
         simulation.get_population_index()
     )
 
     assert isinstance(table, pd.DataFrame)
 
 
+class TestLookupTableResource:
+    @pytest.fixture
+    def manager(self, mocker: MockerFixture) -> LookupTableManager:
+        manager = LookupTableManager()
+        manager.clock = mocker.Mock()
+        manager._pop_view_builder = mocker.Mock()
+        manager._add_resources = mocker.Mock()
+        manager._add_constraint = mocker.Mock()
+        manager._interpolation_order = 0
+        manager._extrapolate = True
+        manager._validate = True
+        return manager
+
+    def test_scalar_table_resource_attributes(self, manager: LookupTableManager) -> None:
+        table = manager._build_table(LookupCreator(), 5, "test_table", value_columns="value")
```

Comment:
what happens if we try and create multiple tables w/ the same name?

Relatedly (I think?), are table ID numbers just gone now in favor of their name?